app/planets_routes.py

```diff
@@ -0,0 +1,138 @@
+from app import db
+from app.models.planet import Planet
+from app.models.moon import Moon
+from flask import Blueprint, jsonify, abort, make_response, request
+from sqlalchemy import desc, asc
+
+
+# ---------helper functions--------
```

Comment:
Specific teams might have their own style guides, but it is a common practice to place helper functions at the end of a file rather than the beginning. An often-used analogy is to lay out code like a newspaper article. Headlines and the most important information comes first - Functions that use other functions in the file go at the top. That gets followed by the details and extra information in an article, or in our code, all the helper functions that support the higher up functions.